### PR TITLE
Owner Transform to Owner

### PR DESCRIPTION
**Pull request by ColdAutumnRain**
**Changelog**
Issue #4451

- changed: Owner.Transform.Coordinates changed to Owner

PS. Sorry for previous PR, for some reason I had file BUILD_TEST inside commit. I wanted to be sure if there are no other file like this.

### CODE REVIEW

File: Content.Server/Atmos/Components/GasTankComponent.cs [hunk 1]
```diff
@@ -283,7 +283,7 @@ public void CheckStatus()
                     if(environment != null)
                         atmosphereSystem.Merge(environment, Air);
 
-                    SoundSystem.Play(Filter.Pvs(Owner), _ruptureSound.GetSound(), Owner.Transform.Coordinates, AudioHelpers.WithVariation(0.125f));
```
Comment (Zumorica) [inline Content.Server/Atmos/Components/GasTankComponent.cs:286]: Given the entity is deleted in the next frame, it makes no sense to play this sound on the owner. Revert the change

File: Content.Server/Body/BodyComponent.cs [hunk 2]
```diff
@@ -104,7 +104,7 @@ public override void Gib(bool gibParts = false)
         {
             base.Gib(gibParts);
 
-            SoundSystem.Play(Filter.Pvs(Owner), _gibSound.GetSound(), Owner.Transform.Coordinates, AudioHelpers.WithVariation(0.025f));
```
Comment (Zumorica) [inline Content.Server/Body/BodyComponent.cs:107]: Given this entity is deleted in the next frame, it makes no sense to play this sound on the owner. Revert the change

File: Content.Server/Movement/Components/FootstepModifierComponent.cs [hunk 3]
```diff
@@ -20,7 +20,7 @@ public class FootstepModifierComponent : Component
 
         public void PlayFootstep()
         {
-            SoundSystem.Play(Filter.Pvs(Owner), SoundCollection.GetSound(), Owner.Transform.Coordinates, AudioParams.Default.WithVolume(-2f));
```
Comment (Zumorica) [inline Content.Server/Movement/Components/FootstepModifierComponent.cs:23]: Honestly, I don't think it makes sense at all that footstep sounds follow an entity. Revert the change

File: Content.Server/Nutrition/Components/SliceableFoodComponent.cs [hunk 4]
```diff
@@ -70,7 +70,7 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
                 }
             }
 
-            SoundSystem.Play(Filter.Pvs(Owner), _sound.GetSound(), Owner.Transform.Coordinates,
```
Comment (Zumorica) [inline Content.Server/Nutrition/Components/SliceableFoodComponent.cs:73]: Play this sound on `eventArgs.User` instead as this entity might be deleted after this.

File: Content.Server/Singularity/Components/ServerSingularityComponent.cs [hunk 5]
```diff
@@ -103,7 +103,7 @@ public void Update(int seconds)
         protected override void OnRemove()
         {
             _playingSound?.Stop();
-            SoundSystem.Play(Filter.Pvs(Owner), _singularityCollapsingSound.GetSound(), Owner.Transform.Coordinates);
+            SoundSystem.Play(Filter.Pvs(Owner), _singularityCollapsingSound.GetSound(), Owner);
```
Comment (metalgearsloth) [inline Content.Server/Singularity/Components/ServerSingularityComponent.cs:106]: In this case the singulo is probably being deleted so probably not suitable to use the entity.

**Review by Zumorica — CHANGES_REQUESTED** [review]

**Comment by Zumorica** [comment]
Two things, just so you know... We don't put technical changes like this in the changelog, users don't care. And your changelog is formatted incorrectly, it needs to follow this format:
```
:cl:
- add: Added fun!
- remove: Removed fun!
- fix:
```

**Review by ShadowCommander — APPROVED** [review]

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**